clouddriver-web/pkg_scripts/rpm/postInstall.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Centos / Red Hat / Amazon Linux
+# check that owner group exists
+if [ -z `getent group spinnaker` ]; then
+  groupadd spinnaker
+fi
+
+# check that user exists
+if [ -z `getent passwd spinnaker` ]; then
+  useradd --gid spinnaker spinnaker -m --home-dir /home/spinnaker
+fi
+
+chkconfig --add clouddriver
```

Comment:
it should be possible to have preInstall and postInstall blocks in `ospackage` that references the common bits (groupadd, useradd, and log directory install) and an additional postInstall in rpm that just adds the calls to chkconfig (those can also be inline in the gradle file instead of externalized to a shell script) e.g.

````groovy
buildRpm {
  postInstall 'chkconfig --add clouddriver'
  postInstall 'chkconfig --level 2345 clouddriver on'
}
````